src/partition/growfs.c

```diff
@@ -49,6 +49,12 @@ static int resize_btrfs(int mountfd, int devfd, uint64_t numblocks, int blocksiz
         struct btrfs_ioctl_vol_args args = {};
         int r;
 
+        /* https://bugzilla.kernel.org/show_bug.cgi?id=118111 */
+        if (numblocks * blocksize < 256*1024*1024) {
+                log_warning("Resizing of btrfs volumes smaller than 256M is not supported");
+                return -EOPNOTSUPP;
```

Comment:
no we log twice about this. Maybe the resize_xyz() calls should log all errors themselves?